templates/admin-followers-list.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
I would not list both tables on one page!

Reply:
I agree! Haven't thought of how to structure this well yet. Any ideas welcome!

Reply:
I would add a tab for that, or a filter. That would be another argument for using the same custom post type as for the follower